host-configs/rzansel-blueos_3_ppc64le_ib_p9-xl@nvcc.cmake

```diff
@@ -96,7 +96,7 @@ set(CMAKE_CXX_COMPILER_ID "XL" CACHE PATH "All of BlueOS compilers report clang
 
 set(BLT_FORTRAN_FLAGS "-WF,-C!  -qxlf2003=polymorphic" CACHE PATH "Converts C-style comments to Fortran style in preprocessed files")
 
-set(BLT_EXE_LINKER_FLAGS "-Wl,-rpath,/usr/tce/packages/xl/xl-2019.08.20/lib" CACHE PATH "Adds a missing rpath for libraries associated with the fortran compiler")
+set(BLT_EXE_LINKER_FLAGS "-Wl,-rpath,/usr/tce/packages/xl/xl-2019.08.20/lib -L/usr/tce/packages/cuda/cuda-10.1.168/lib64" CACHE PATH "Adds a missing rpath for libraries associated with the fortran compiler")
 
```

Comment:
Although this works, I think it would be best if BLT does that in `blt_setup_cuda_target()`, e.g., by adding a `LINK_DIRECTORIES` target property to the specified target. Setting `BLT_EXE_LINKER_FLAGS` is handy but, it seems like a large hammer.

I'll look more into BLT and talk to a few folks -- Let me know what you think.